support-models/src/test/scala/com/gu/support/promotions/SerialisationSpec.scala

```diff
@@ -69,6 +69,17 @@ class SerialisationSpec extends FlatSpec with SerialisationTestHelpers with Lazy
     )
   }
 
+  it should "be able to decode an introductory price Promotion" in {
+    testDecoding[Promotion](Fixtures.introductoryPricePromotion,
+      promotion => {
+        promotion.introductoryPrice.isDefined shouldBe true
```

Comment:
nitpick: `should be (defined)` might give a better fail message on this assertion.